src/deletepy/utils/legacy_print.py

```
@@ -0,0 +1,224 @@
+"""Legacy print function replacements with logging.
+
+This module provides drop-in replacements for the existing print-based
+functions to gradually migrate to structured logging while maintaining
+backward compatibility.
+"""
+
+import logging
```

Ruff (0.11.9)

`logging` imported but unused

Remove unused import: `logging`

(F401)